eval/src/vespa/eval/eval/aggr.h

```diff
@@ -170,7 +170,7 @@ template <typename T> class Median {
     constexpr Median() noexcept : _seen() {}
     constexpr Median(T value) : _seen({value}) {}
     constexpr void sample(T value) { _seen.push_back(value); }
-    constexpr void merge(const Median &rhs) {
+    void merge(const Median &rhs) {
```

Comment:
This feels like the wrong solution. I assume this is due to not having noinline on m_insert_aux.

Reply:
I suggest we try to come up with another solution.

Reply:
m_insert_aux is constexpr. https://gcc.gnu.org/bugzilla/show_bug.cgi?id=97681 indicates that noinline attribute might be ignored.